examples/common/resumption_callbacks/callbacks.py

```diff
@@ -0,0 +1,50 @@
+from composer.core import Callback, State
+from composer.loggers import Logger
+from typing import List
+
+__all__ = ['ResumptionCallback', 'GlobalLRScaling', 'LayerFreezing',]
+
+
+class ResumptionCallback(Callback):
```

Comment:
do we want to label this as a AbstractBaseClass

Reply:
good idea!